codex/sales.nim

```diff
@@ -108,6 +108,7 @@ proc remove(sales: Sales, agent: SalesAgent) {.async.} =
 
 proc cleanUp(sales: Sales,
              agent: SalesAgent,
+             returnBytes: bool,
```

Comment:
In what cases do we want this to be false? Shouldn't we always try to return bytes to repo?

Reply:
In the case where we successfully finish the state machine, in the current state, we don't have "persistent availabilities", so we don't return bytes once the sale is successfully finished. I created an issue about it here #467 some time ago. We can implement that afterwards, but this PR is not about that.